packages/i18n/lib/set_related_locale_import.ts

```diff
@@ -0,0 +1,39 @@
+import { lstatSync, readdirSync, readFileSync, writeFileSync } from 'node:fs';
+import { resolve } from 'node:path';
+import type { SupportedLanguagesKeysType, SupportedLanguagesWithoutRegionKeysType } from './types.js';
+import { I18N_FILE_PATH } from './consts.js';
+
+const CEB_DEV_LOCALE = 'en';
```

Comment:
It seems like TODO

Reply:
Yeah, it was placeholder till I'll merge #725 
That's why i've changed branch to `dev`

Reply:
https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/711#discussion_r1914723365

Same